cmd/kots/cli/garbage-collect-images.go

```diff
@@ -0,0 +1,126 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/pkg/auth"
+	"github.com/replicatedhq/kots/pkg/k8sutil"
+	"github.com/replicatedhq/kots/pkg/logger"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func GarbageCollectImagesCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "garbage-collect-images [namespace]",
+		Short:         "Run image garbage collection",
+		Long:          `Triggers image garbage collection for all apps`,
+		SilenceUsage:  true,
+		SilenceErrors: false,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			viper.BindPFlags(cmd.Flags())
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			v := viper.GetViper()
+
+			log := logger.NewCLILogger()
+
+			// use namespace-as-arg if provided, else use namespace from -n/--namespace
+			namespace := v.GetString("namespace")
+			if len(args) == 1 {
+				namespace = args[0]
+			} else if len(args) > 1 {
+				fmt.Printf("more than one argument supplied: %+v\n", args)
+				os.Exit(1)
+			}
+
+			if namespace == "" {
```

Comment:
i would suggest using this function instead https://github.com/replicatedhq/kots/blob/051058090f62305445d9761930eb9b0775839774/cmd/kots/cli/namespace.go#L10